docs/modules/ROOT/nav.adoc

```diff
@@ -2,7 +2,7 @@
 * xref:README.adoc[Introduction]
 ** xref:general-info/general-info.adoc[Overview]
 ** xref:release-notes/README.adoc[Release Notes]
-*** xref:release-notes/release-notes-23-0.adoc[Payara Enterprise 5.23.0 Release Notes]
```

Comment:
Please keep the link to RN of 23.o here too. We're not going to release a completely new version of the docs, we'll just update existing 5.23 docs. The pattern we use is that we just add a new page for each patch version. So there will be 2 RN pages here now: for 5.23.0 and for 5.23.1

Reply:
@cubastanley , I think you misunderstood me. There should be now both release notes for 5.23.0 and 5.23.1 linked here. Similar to this PR for 5.21.2: https://github.com/payara/Payara-Enterprise-Documentation/pull/48/files#diff-2074dbd9f25b3a4b6f9d1f2ce74c297bd9c97be43683b94ee7c7a8e3eb6c3d6fR5. We always include RN for the latest minor version and all its patch versions here.